compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaLexer.java

```diff
@@ -791,6 +791,10 @@ private STToken processIdentifierOrKeyword() {
         }
 
         String tokenText = getLexeme();
+        if (isPredeclaredPrefix(tokenText) && peek() == ':') {
```

Comment:
Shall keep the lexer unchanged and only change the parser?

Reply:
Done